change/@ni-nimble-components-163379f4-09c6-4606-95f4-05917dd50ee0.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Created new hover event for the new diesTable api and changed the zoom event",
+  "packageName": "@ni/nimble-components",
+  "email": "33986780+munteannatan@users.noreply.github.com",
+  "dependentChangeType": "patch"
+}
```

Comment:
There are not any package.json changes in this PR so I wouldn't expect any lock file changes.
May need to bring in the lock file from main